src/components/AddressSearch/index.js

```diff
@@ -382,6 +389,21 @@ function AddressSearch(props) {
                                 />
                             </View>
                         }
+                        renderRow={(data) => {
+                            const title = data.isPredefinedPlace ? data.name : data.structured_formatting.main_text;
```

Comment:
Would we have any case where `title` could be empty?

Reply:
@mananjadhav I don't think so because we I have made sure to set it to a value by default and it should be set on the backend as well...

Reply:
Thanks for clarifying. One last comment on the storybook.

Reply:
Replied here: https://github.com/Expensify/App/pull/29931#issuecomment-1771503398